zeppelin-interpreter/src/main/java/org/apache/zeppelin/helium/HeliumPackage.java

```diff
@@ -18,13 +18,16 @@
 
 import org.apache.zeppelin.annotation.Experimental;
 
+import java.util.Map;
+
 /**
  * Helium package definition
  */
 @Experimental
 public class HeliumPackage {
   private HeliumType type;
   private String name;           // user friendly name of this application
+  private String version;
```

Comment:
Thanks for detailed review. 

The reason I added `version` field is

- it's hard to get version field when package is local. In this case artifact doens't include version value at all. 

I thought having the duplicated field is more cheap and simple way to implement instead of split logic to support both cases (local, remote)